package.json

```diff
@@ -56,7 +56,7 @@
   },
   "devDependencies": {
     "@types/node": "^12.0.2",
-    "ava": "^0.25.0",
+    "ava": "^3.15.0",
```

Comment:
I don't think this needs bumping. I wouldn't mind either way, except that this jumps through quite a few breaking changes without the necessary updates. If you run `npm test` (on any platform) you'll find that this breaks all the tests, because Ava 3 works very differently to Ava 0.25.

Sorry, that should've been detected by CI here when you opened the PR, but tests weren't run automatically for PRs - I've now [fixed that](https://github.com/httptoolkit/browser-launcher/commit/396005682f592fd8ad3bf460d19499d0c04d8d32) so new pushes should be tested correctly.